packages/salesforcedx-utils-vscode/src/helpers/utils.ts

```diff
@@ -14,11 +16,28 @@ export function isNullOrUndefined(object: any): object is null | undefined {
 }
 
 export function extractJsonObject(str: string): any {
-
   const jsonString = str.substring(
     str.indexOf('{'),
     str.lastIndexOf('}') + 1
   );
 
   return JSON.parse(jsonString);
 }
+
+// There's a bug in VS Code where, after a file has been renamed,
+// the URI that VS Code passes to the command is stale and is the
+// original URL.  See https://github.com/microsoft/vscode/issues/152993.
+//
+// To get around this, fs.realpathSync.native() is called to get the
+// URI with the actual file name.
+export function flushFilePath(filePath: string): string {
```

Comment:
Nit: Naming is hard.  I'm ok with flushFilePath, but would getCaseSenstiveFilePath be more accurate as far as what we're trying to accomplish?

Reply:
I don't think it wold be more accurate, because the path is already case sensitive, and the issue is that it is old & stale, and needs to be updated.  It took me a while to land on `flushFilePath`, just b/c I couldn't think of a name that's more fitting or more accurately described the issue or what it does.

But that being said, I'm OK with the current name but also not totally in love with it either.  I don't think it should be `getCaseSenstiveFilePath`, but if you have any other suggestions, please let me know.